modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/model/impl/DDMFormInstanceImpl.java

```diff
@@ -58,6 +58,15 @@ public DDMFormInstanceVersion getFormInstanceVersion(String version)
 			getFormInstanceId(), version);
 	}
 
+	@Override
+	public Long getObjectDefinitionId() throws PortalException {
+		DDMFormInstanceSettings ddmFormInstanceSettings =
+			DDMFormInstanceFactory.create(
+				DDMFormInstanceSettings.class, getSettingsDDMFormValues());
+
+		return Long.valueOf(ddmFormInstanceSettings.objectDefinitionId());
+	}
```

Comment:
@mlan0 Why did you choose to use `DDMFormInstanceFactory.create` instead of `DDMFormInstanceSettings ddmFormInstanceSettings = getSettingsModel()`?

Reply:
In addition, can you change the return type to `long` and use `GetterUtil.getLong` instead of `Long.valueOf`? Thanks!

Reply:
@carolmariaabb I followed the pattern used in the `getStorageType()` method as it has similar functionality.

Reply:
Got it! They both do the same thing, so let's keep it consistent with the `getStorageType()` method.
Gret job, @mlan0 👏🏽